core/domain/customization_args_util_test.py

```diff
@@ -44,7 +46,9 @@ def test_get_full_customization_args(self):
             }
         }
 
-        complete_customization_args_with_extra_arg = {
+        complete_customization_args_with_extra_arg: (
+            Dict[str, Dict[str, Union[str, Dict[str, Optional[str]]]]]
```

Comment:
Can we define a custom type for this?

Reply:
Ditto elsewhere.

Reply:
I think we should not do this because these dictionaries are defined just for testing purposes and only used one time in the test file.